server/survey.go

```diff
@@ -14,16 +14,19 @@ import (
 const (
 	// How often "survey scheduled" emails can be sent to prevent multiple emails from being sent if multiple server
 	// upgrades occur within a short time
-	MIN_DAYS_BETWEEN_SURVEY_EMAILS = 7
+	MIN_TIME_BETWEEN_SURVEY_EMAILS = 7 * 24 * time.Hour
```

Comment:
Not really part of this PR... but shouldn't these be config settings?

Reply:
I don't think we wanted them to be configurable since it wasn't included in the spec. I don't know if it just wasn't thought about at the time, or if there was some reason for not including them like not letting admins increase the rate that the surveys came out so it would annoy their users.